Scripts/ros.patch

```diff
@@ -0,0 +1,26 @@
+Index: Source/ThirdParty/ros/include/rcutils/error_handling.h
+IDEA additional info:
+Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
+<+>UTF-8
+===================================================================
+diff --git a/Source/ThirdParty/ros/include/rcutils/error_handling.h b/Source/ThirdParty/ros/include/rcutils/error_handling.h
+--- a/Source/ThirdParty/ros/include/rcutils/error_handling.h	(revision 584767ce5b22125a3cffe1e9e84618068f1431b4)
++++ b/Source/ThirdParty/ros/include/rcutils/error_handling.h	(date 1652721130821)
+@@ -88,7 +88,7 @@
+ } rcutils_error_state_t;
+ 
+ // make sure our math is right...
+-#if __STDC_VERSION__ >= 201112L
```

Comment:
What do you mean? I created this path with macro removing, because same patching was before my changes and was applyed to include folder. Also once in a while, not always, it helps to fix build at compile time

Reply:
Ah, I did not realize this patch was already here.